Build/Scripts/install_tools.ps1

```diff
@@ -0,0 +1,82 @@
+# Put together from script originally by JamesNK
+# https://github.com/JamesNK/Newtonsoft.Json/blob/c89d6addf118745c4c14536ce64fd69566ebd644/Build/build.ps1
+$ErrorActionPreference = "Stop"
+
+$nugetUrl = 'https://dist.nuget.org/win-x86-commandline/latest/nuget.exe'
+$nugetPath = '.\Temp\nuget.exe'
+$nugetSource = 'https://api.nuget.org/v3/index.json'
+$vswhereVersion = '2.3.2'
+$vswherePath = ".\Temp\vswhere.$vswhereVersion"
+$nunitConsoleVersion = '3.8.0'
+$nunitConsolePath = ".\Temp\NUnit.ConsoleRunner.$nunitConsoleVersion"
+
+function Install-AllTheThingsINeed()
+{
+    New-Item Temp -ItemType Directory -ErrorAction SilentlyContinue
+    EnsureNuGetExists
+    EnsureNuGetPackage "vswhere" $vswherePath $vswhereVersion
+    EnsureNuGetPackage "NUnit.ConsoleRunner" $nunitConsolePath $nunitConsoleVersion
+
+    $msBuildPath = GetMsBuildPath
+    Write-Host "MSBuild path $msBuildPath"
+
+@"
+`$ErrorActionPreference = "Stop"
+`$nuget = '$(Resolve-Path $nugetPath)'
+`$msbuild = '$msBuildPath'
+`$vswhere = '$(Resolve-Path $vswherePath\tools\vswhere.exe)'
+`$nunit3console = '$(Resolve-Path $nunitConsolePath\tools\nunit3-console.exe)'
+"@ > .\Temp\profile.ps1
+}
+
+function EnsureNuGetExists()
+{
+    if (!(Test-Path $nugetPath))
+    {
+        Write-Host "Couldn't find nuget.exe. Downloading from $nugetUrl to $nugetPath"
+        $ProgressPreference = "SilentlyContinue"
+        Invoke-WebRequest -Uri $nugetUrl -OutFile $nugetPath
+    }
+    else
+    {
+        Write-Host "nuget.exe was already installed."
+    }
+}
+
+function EnsureNuGetPackage($packageName, $packagePath, $packageVersion)
+{
+    if (!(Test-Path $packagePath))
+    {
+        Write-Host "Couldn't find $packagePath. Downloading with NuGet"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File 'install_tools.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5097765)